fedcred/common.py

```diff
@@ -49,13 +49,19 @@ def read_config():
 
 
 def get_user_credentials(prompt=None):
-    if prompt is None:
+    if "AWS_USERNAME" in os.environ:
+        username = os.environ["AWS_USERNAME"]
+    elif prompt is None:
         prompt_msg = 'Enter you username: '
-    try:
-        username = raw_input(prompt_msg).strip()
-    except NameError:
-        username = input(prompt_msg).strip()
-    password = getpass.getpass(prompt='Enter your passowrd: ')
+        try:
+            username = raw_input(prompt_msg).strip()
+        except NameError:
+            username = input(prompt_msg).strip()
+
+    if "AWS_PASSWORD" in os.environ:
```

Comment:
Same as previous comment. Also, I wouldn't recommend storing your password in an environment variable.